src/Symfony/Component/Workflow/Workflow.php

```diff
@@ -59,10 +59,6 @@ public function getMarking($subject)
     {
         $marking = $this->markingStore->getMarking($subject);
 
-        if (!$marking instanceof Marking) {
```

Comment:
I agree this should not be needed though. Otherwise it would mean that the marking store is violating the interface contract made [here](https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Workflow/MarkingStore/MarkingStoreInterface.php#L28) and [here](https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Workflow/Workflow.php#L32). Does that make sense ?